### ISSUE TITLE
v-repeat with v-component can't use replace:true in some cases

### ISSUE THREAD

**Issue by OEvgeny** [yyx990803/vue#820]
I'm noticed what can't use ``paramAttributes`` on v-repeated component. And it is frustrating me.
Fortunately found solution (see https://github.com/yyx990803/vue/issues/816), but it is not working with ``replace:true`` for child component.
See fiddle: http://jsfiddle.net/rynrj5Ls/2/ and try to comment ``replace:true``

Got error:
```
Uncaught InvalidCharacterError: Failed to execute 'setAttribute' on 'Element': '<' is not a valid attribute name.
```

On my local project got some kind of error like:
```
Uncaught TypeError: this.el.setAttribute is not a function
```
Because of ``this.el`` is ``DocumentFragment`` and hasn't ``setAttribute`` method.

Is it "by design", and will it change in feature? Think what it prevents to develop fully independent components.

Also thank you for a great work.

**Comment by azamat-sharapov** [comment]
In your example, if you remove `replace: true` and just use following template for main vm instance, it works:

```
<template v-repeat="items"> <ui-test first="{{ outer }}" second="{{ inner }}"</ui-test></template>
```

Doesn't it work for your situation?

**Comment by azamat-sharapov** [comment]
Oh, actually `replace: true` works. You just have a typo in your template. It should be:

```
<div v-repeat="items"> <ui-test first="{{ outer }}" second="{{ inner }}"></ui-test></div>
```

**Comment by OEvgeny** [comment]
Yeah, jsfiddle now works!
Now trying to find cause of my local bug.
Thank you alot!

**Comment by azamat-sharapov** [comment]
For your second issue - try `this.$el.setAttribute()` instead.

**Comment by OEvgeny** [comment]
Can't do it because ``setAttribute()`` is called by Vue in internals.

**Comment by azamat-sharapov** [comment]
well, then show some code :)

**Comment by sjoerdvisscher** [comment]
Is this related to #821?

**Comment by OEvgeny** [comment]
http://jsfiddle.net/rynrj5Ls/3/
Now figured out why. No root tag on the template.
May be we need a warning in this case on debug mode?

**Comment by OEvgeny** [comment]
@sjoerdvisscher no, my fiddle uses 0.11.8 as I see.

**Comment by azamat-sharapov** [comment]
You are telling vue to `replace` the `ui-test` tag, but asking vue to output v-repeat's item as a attribute value of `ui-test` tag. I think error is expected in this case of course..

**Comment by OEvgeny** [comment]
Are you think warning on debug mode not needed in this case?

**Comment by azamat-sharapov** [comment]
Warning message would definetely be helpful, you are right. Let's hear from @yyx990803, he will decide, if available.

**Comment by yyx990803** [comment]
This actually works correctly in 0.11.9 ;)
But there was another regression (#822) which I just fixed, so expect 0.11.10 soon

**Comment by OEvgeny** [comment]
Oh thanks Evan!
